definitions/ext-rf_scanner/summary_metrics.yml

```
@@ -44,10 +9,25 @@ totalTime:
     eventId: entity.guid
     
 actions:
-  title: Actions 
+  title: Volume 
   unit: COUNT
   query:
-    select: count(action)
-    from: THD_RF_CONNECTION_EVENT
+    select: count(*)
+    from: THD_RF_TRANSACTION
     eventId: entity.guid
-    
+
+errors:
+  title: Errors
+  unit: COUNT
+  query:
+    select: count(message)
+    from: Log
+    where: message like '%error%' or message like '%Error%' and message NOT LIKE '%lqqqq%' or message NOT LIKE '%Winsock error 10042%'
+
```

```suggestion
eventId: entity.guid

```

The PR is failing because `eventId` is mandatory here